text/0000-export-to-oci.md

```diff
@@ -252,22 +252,16 @@ I tried different approaches, the best results were found when using the [stream
 
 | Sample  | Image size (MB)|  First Build |  Second Build |
 |---|---|---|---|
-| Java  | 238.1  | 2.41s † <br> 4.76s ‡ (+10.96%) <br> 3.21 ⁜ <br> =10.38s | - |
-| Kotlin  | 305.98 | 2.39s † <br> 4.73s ‡ (+10.77%) <br> 3.38s ⁜ <br> =10.5s  | -  |
-| Ruby  | 100.52  |  2.59s † <br> 1.31s ‡ (+13.91%) <br> 0.89s ⁜ <br> =4.79s  | -  |
+| Java  | 238.1  | 2.41s † <br> 4.76s ‡ (+10.96%) <br> 3.21 ⁜ <br> =10.38s | 3.47s † <br> 4.98s ‡ (+55.63%) <br> 3.02 ⁜ <br> = 11.47s|
+| Kotlin  | 305.98 | 2.39s † <br> 4.73s ‡ (+10.77%) <br> 3.38s ⁜ <br> =10.5s  | 3.47s † <br> 5.11s ‡ (+61.20%) <br> 3.22s ⁜ <br> = 11.8s  |
+| Ruby  | 100.52  |  2.59s † <br> 1.31s ‡ (+13.91%) <br> 0.89s ⁜ <br> =4.79s  | 2.44s † <br> 1.51s ‡ (+122.06%) <br> 1.13s ⁜ <br> = 5.08s  |
```

Comment:
It's interesting to note that second builds are actually slower than the first. I think this must be due to the re-uploading of ALL the layers vs. just the changed ones (the imgutil hack).

Reply:
Ah, I should have been more clear - for second builds, I was expecting (if there aren't many changed layers) that ⁜ times would be significantly faster.

Reply:
I think the new table summarized better the results